spec/fixtures/list.xml

```diff
@@ -0,0 +1,1396 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<GoodreadsResponse>
+   <Request>
+      <authentication>true</authentication>
+      <key><![CDATA[GFPTphT7xVUhrarWQztUtg]]></key>
```

Comment:
Also, currently working on the travis ci test issue. Trying to see if changing the version num of active support will fix it.